### PR TITLE
ENH Replace ADMIN permissions with less permissions in Behat test

### PR DESCRIPTION
**Pull request by sabina-talipova**
### Description
Replace user with "Admin" permissions to user with "Editor" permissions.

### Parent issue
- https://github.com/silverstripe/silverstripe-contentreview/issues/171

### CODE REVIEW

**Comment by GuySartorelli** [comment]
Tests still failing

File: tests/behat/features/content-reviews.feature [hunk 1]
```diff
@@ -39,6 +43,8 @@ Feature: Set up reviews
     When I press the "Save" button
     And I select "ADMIN group" from "Groups"
     And I press the "Save" button
+    And I select "EDITOR" from "Groups"
+    And I press the "Save" button
```
Comment (ChloeSartorelli) [inline tests/behat/features/content-reviews.feature]: I don't understand why we need to save two groups here?

Comment (sabina-talipova) [reply tests/behat/features/content-reviews.feature]: We have a button `.content-review__button` that appears only if user has special permission. So I add `EDITOR` to check this button in the same case.

Comment (sabina-talipova) [reply tests/behat/features/content-reviews.feature]: I removed `ADMIN group`.

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**